backend/oas/admin/schemas/user.yaml

```diff
@@ -0,0 +1,32 @@
+user.GetOneUserResponse:
```

Comment:
Please adjust the coding style to match that of the schema files for the user/provider API. For example, double quotation marks around values like `type: "integer"` are unnecessary and not used in those files.

Additionally, it seems that files are split too finely; `users.yaml` and `user.yaml` could be combined into a single `user.yaml` file, because these two files are describing a single resource: `users`. However, if I have misunderstood the coding guidelines, please let me know -- @orangekame3

Reply:
> Additionally, it seems that files are split too finely; users.yaml and user.yaml could be combined into a single user.yaml file, because these two files are describing a single resource: users. However, if I have misunderstood the coding guidelines, please let me know

Yes, it should be same file.
The granularity of the files has been aligned with that of the resource (tag).